### DISCUSSION TITLE
9.38依旧无法联机

### DISCUSSION THREAD

**Discussion by pengpai1027** [IndustrialCraftMC/IndustrialCraft2#147] [【漏洞反馈】Bugs Report]
还是老问题
![114514](https://github.com/IndustrialCraftMC/IndustrialCraft2/assets/159503942/74e6e1c5-bd36-45dc-962d-f8f9e0009b24)

**Comment by Alirea10** [comment]
![2bd659307ca9f1a6c134e481b443f2e4](https://github.com/IndustrialCraftMC/IndustrialCraft2/assets/111880868/d374a4e5-a7c5-45ca-bf17-a5590c95dae8)
此图疑似也是一个问题。

**Comment by Alexme9125** [comment]
<img width="1707" alt="Screenshot 2024-02-11 at 17 24 28" src="https://github.com/IndustrialCraftMC/IndustrialCraft2/assets/87228326/f1ac027c-488e-4dc8-8bac-710a3ce632b9">
在Forge端的服务器中仍然出现此类问题。
服务器正常运行，但问题发生在玩家正在加入时。
运行硬件为Apple M1 Max，平台为macOS 14.4 Developer Beta 2。（服务器在Ubuntu上运行也会出现一样的问题）
